app/views/standup_meeting_groups/index.html.erb

```diff
@@ -6,14 +6,14 @@
 
 <div class="flex flex-col justify-center">
   <h2 class="font-bold text-3xl pl-5 text-center md:text-left">My Standups</h2>
-  <div id="my-standup-meeting-groups" class=" flex justify-center flex flex-wrap gap-5 pl-5 pt-5 pb-5">
+  <div id="my-standup-meeting-groups" class="flex flex-wrap gap-5 pl-5 pt-5 pb-5">
     <%= render StandupMeetingGroup::JoinedCardComponent.with_collection(@my_standup_meeting_groups, current_user:) %>
   </div>
 </div>
 
 <div class="flex flex-col">
  <h2 class="font-bold text-3xl pl-5 text-center md:text-left">Find a Standup</h2>
   <ul id="find-standup-meeting-groups" class="flex flex-col pl-5 pt-5 pb-5">
```

Comment:
Not a blocker, and not even something your changes are responsible for, but before merge, I think it'd be cool to adjust the padding here so the Joinable cards look a bit better on mobile.

![Selection_247](https://github.com/agency-of-learning/PairApp/assets/88392688/8339145d-876a-4d5c-a04d-e58f35716336)